ios/versioned/sdk43/EXUpdates/EXUpdates/ReactDelegateHandler/ExpoUpdatesReactDelegateHandler.swift

```diff
@@ -1,110 +0,0 @@
-// Copyright 2018-present 650 Industries. All rights reserved.
-
-import ABI43_0_0ExpoModulesCore
-
-public class ExpoUpdatesReactDelegateHandler: ExpoReactDelegateHandler, ABI43_0_0EXUpdatesAppControllerDelegate, ABI43_0_0RCTBridgeDelegate {
-  private weak var reactDelegate: ExpoReactDelegate?
-  private var bridgeDelegate: ABI43_0_0RCTBridgeDelegate?
-  private var launchOptions: [AnyHashable: Any]?
-  private var deferredRootView: ABI43_0_0EXDeferredRCTRootView?
-  private var rootViewModuleName: String?
-  private var rootViewInitialProperties: [AnyHashable: Any]?
-  private lazy var shouldEnableAutoSetup: Bool = {
```

Comment:
Is this file safe to remove because `shouldEnableAutoSetup` is always false in Expo Go?

Reply:
This file is safe to remove since it is unused (and has compilation issues).